### DISCUSSION TITLE
User Additional field not saved to database

### DISCUSSION THREAD

**Discussion by golatam** [orchidsoftware/platform#1903] [Q&A] [answered]
Hi everyone!

I have an problem.
For my project i added new field for User table, also i changed it field to fillable:
`    protected $fillable = [
        'name',
        'email',
        'password',
        'permissions',
        'tosname',
    ];`

I seen that this field included to array for update/save, but isn't saved to database.

Example:

```
$user
->fill($userData)
 ->fill([
 'permissions' => $permissions,
 ]) ->save();
dd($userData);
```

And the result:

`array:3 [▼
  "name" => "admin2"
  "email" => "tosadmin@init-rk.ru"
  "tosname" => "asdasd"
]`

But didn't save it on database:

![image](https://user-images.githubusercontent.com/91856472/135778744-f8c7188e-870c-49a9-933a-609f1bda5a9a.png)

Can you help me? Thanks a lot!

**Comment by tabuna** [comment, marked as answer]
Pay attention to the full path of the model specified in the file on the screen. I think you need to change it:

https://github.com/orchidsoftware/platform/blob/e4cc3ec4693001a54429d0c98ad28d76c1bb8449/stubs/app/Orchid/Screens/User/UserEditScreen.php#L15

**Comment by golatam** [comment]
Thanks a lot!
Really working right now!
